<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FcitxWarnDialog</class>
 <widget class="QDialog" name="FcitxWarnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="btn_min">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>5</y>
      <width>26</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="btn_close">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>26</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>331</width>
     <height>160</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>15</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tips:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>120</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="displaylabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>49</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/message.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pixmap.qrc">:/pixmap/image/message.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pixmap.qrc"/>
 </resources>
 <connections/>
</ui>
